packages/cli/src/PublicApi/v1/Credentials/credentials.handler.ts

```diff
@@ -0,0 +1,90 @@
+import express = require('express');
+
+import { CredentialsEntity } from '../../../databases/entities/CredentialsEntity';
+import { CredentialRequest } from '../../../requests';
+import { externalHooks } from '../../../Server';
+
+import {
+	createCredential,
+	encryptCredential,
+	getCredentials,
+	getSharedCredentials,
+	removeCredential,
+	sanitizeCredentials,
+	saveCredential,
+} from './credentials.service';
+
+export = {
+	createCredential: [
+		async (
+			req: CredentialRequest.Create,
+			res: express.Response,
+		): Promise<express.Response<Partial<CredentialsEntity>>> => {
+			delete req.body.id; // delete if sent
```

Comment:
As I mentioned bellow, I would error if something that is not in the spec is sent